### DISCUSSION TITLE
Adding metrics in the time series package

### DISCUSSION THREAD

**Discussion by brunobarella** [pycaret/pycaret#3068] [Q&A]
What is the correct way to create metrics functions for the time series? And avoid NaN values in create_model and compare_models methods?
![image](https://user-images.githubusercontent.com/28939189/200141053-7f77a8c7-6f24-4d1c-9cbf-771f24990d2c.png)

**Comment by ngupta23** [comment]
Please check the closed issues/discussion/FAQ's in GitHub. This has been answered before. Thanks!
